pwndbg/commands/misc.py

```diff
@@ -96,29 +92,17 @@ def errno_(err) -> None:
 )
 
 
-@pwndbg.commands.ArgparsedCommand(parser, command_name="pwndbg", category=CommandCategory.PWNDBG)
-def pwndbg_(filter_pattern, shell, all_, category_, list_categories) -> None:
+@pwndbg.commands.Command(parser, command_name="pwndbg", category=CommandCategory.PWNDBG)
+def pwndbg_(filter_pattern, category_, list_categories) -> None:
     if list_categories:
         for category in CommandCategory:
             print(C.bold(C.green(f"{category.value}")))
         return
 
-    if all_:
```

Comment:
Just for future: ideally this should be in another PR, but we can leave it in this one.